pkgs/development/python-modules/skops/default.nix

```
@@ -0,0 +1,70 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  setuptools,
+  pytestCheckHook,
+  huggingface-hub,
+  matplotlib,
+  pandas,
+  scikit-learn,
+  streamlit,
+  tabulate,
+}:
+
+buildPythonPackage rec {
+  pname = "skops";
+  version = "0.9.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "skops-dev";
+    repo = "skops";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-JCK5x2OKRcUftld+bGgHD74eB3Jmij2PZnOPIenTRlM=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml  \
+      --replace-fail "--cov=skops" ""  \
+      --replace-fail "--cov-report=term-missing" ""
+  '';
+
+  build-system = [
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
```

```suggestion
  dependencies = [
```
It's better to use `dependencies` to conform `pyproject.toml`.

---

Done.